client/my-sites/themes/helpers.js

```diff
@@ -104,6 +103,11 @@ export function getExternalThemesUrl( site ) {
 }
 
 export function trackClick( componentName, eventName, verb = 'click' ) {
+	// The analytics module is highly reliant on BOM and DOM, and while we can mock
+	// the entire module trivially on server build, we can't do the same for test
+	// builds, so importing it at the top level broke tests. This is why we're using
```

Comment:
Oh, this change is actually moot since I removed the simple test that imported this module. I'll revert, but that's likely a better solution if this module gets tests in the future.